src/pages/index.jsx

```diff
@@ -542,7 +542,7 @@ function Title() {
         <div className='justify-center mt-10 flex flex-wrap items-center gap-x-6 gap-y-8'>
           <Button variant='juicy'>Sign Up for Beta</Button>
 
-          <Link href='/learn' className='text-sm font-semibold leading-6 text-white'>
+          <Link href='/learn' className='text-sm font-semibold leading-6 text-white hover:after:content-[""] hover:after:block hover:after:w-full hover:after:h-1 hover:after:bg-rainbow-gradient' >
```

Comment:
Fix the shifting text with:

```suggestion
          <Link href='/learn' className='text-sm font-semibold leading-6 text-white after:content-[""] after:block after:w-full after:h-1 after:bg-rainbow-gradient after:opacity-0 hover:after:opacity-100' >
```

Reply:
Might want to add a negative bottom margin to make it align.